src/libs/ValidationUtils.js

```diff
@@ -41,7 +44,10 @@ function isRequiredFulfilled(value) {
  * @returns {Boolean} true if valid
  */
 function isValidDate(date) {
-    return moment(date).isValid();
+    const pastDate = moment().subtract(1000, 'years');
+    const futureDate = moment().add(1000, 'years');
+    const testDate = moment(date);
+    return testDate.isValid() && testDate.isBetween(pastDate, futureDate);
```

Comment:
@roryabraham I added this to catch this problem before sending it to the API (see [here](https://github.com/Expensify/Web-Secure/blob/d4ce70cca024cfa55adef2ec774ae9f9d3a526e6/lib/User/Utils.php#L460-L471))